tests/phpunit/CRM/Event/BAO/CRM19273Test.php

```diff
@@ -81,7 +81,6 @@ protected function cleanup() {
    * @return int
    */
   protected function eventPriceSetCreate($amount = 0, $min_fee = 0) {
-
```

Comment:
All these changes in the test were to get it working - turns out it didn't originally run so we didn't know it didn't work :-(